src/gluonts/trainer/_base.py

```diff
@@ -147,19 +151,18 @@ def __init__(
         self.init = init
         self.hybridize = hybridize
         self.ctx = ctx if ctx is not None else get_mxnet_context()
-        self.halt = False
```

Comment:
It’s fine to remove this, since it’s only used for interactively breaking the loops; however, we probably want to have some other early stopping mechanism (maybe through the learning rate scheduler, as soon as the minimum lr is hit or exceeded?) so something will need to be there